config/checkstyle/checkstyle.xml

```diff
@@ -143,7 +143,7 @@
 		<module name="TrailingComment">
 			<property name="legalComment" value="NOSONAR"/>
 		</module>
-		<module name="TodoComment" />
+		<!--<module name="TodoComment" />-->
```

Comment:
I think we'd want to keep that to be consistent with Atlas, and instead use `// TODO` in place of `// @TODO` which triggers it.

Reply:
oh, okay! I can fix that